.pre-commit-config.yaml

```diff
@@ -1,21 +1,20 @@
 repos:
   - repo: local
     hooks:
-      - id: ruff
-        name: ruff
-        entry: ruff check .
-        args: [ --config, pyproject.toml, --fix, --show-fixes, --exit-non-zero-on-fix ]
+      - id: ruff-lint
+        name: ruff-lint
+        entry: ruff check
+        args: [ --force-exclude, --config, pyproject.toml, --fix, --show-fixes, --exit-non-zero-on-fix ]
         language: system
         types_or: [python]
         require_serial: true # run once for all files
-        pass_filenames: false
-      - id: black
-        name: black
-        entry: black
+      - id: ruff-format
+        name: ruff-format
+        entry: ruff format
+        args: [ --force-exclude, --config, pyproject.toml ]
         language: system
-        types_or: [python, pyi]
+        types_or: [python]
         require_serial: true # run once for all files
-        exclude: ^tests/.*snapshots/
```

Comment:
don't we need to continue excluding the snapshots? it's unnecessary to format them since they're generated

Reply:
They are excluded in `pyproject.toml`